src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int DEFAULT_RADIUS = 10;
+    public static final int FIGURE_COUNT = 5;
+    public static final int UPPER_BOUND = 31;
+
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    
+    public Figure getRandomFigure() {
+        int figureIndex = random.nextInt(FIGURE_COUNT);
+        Figure figure;
+        switch (figureIndex) {
+            case 0: 
+                int randomRadius = random.nextInt(UPPER_BOUND);
+                figure = new Circle(colorSupplier.getRandomColor(), randomRadius);
+                break;
```

Comment:
```suggestion
        int figureIndex = random.nextInt(FIGURE_COUNT);
        switch (figureIndex) {
            case 0: 
                return getRandomCircle();
```
lets make private methods to get each random figure to make your code easier to read